gym/__init__.py

```diff
@@ -1,23 +1,25 @@
 """Root __init__ of the gym module setting the __all__ of gym modules."""
 # isort: skip_file
+import os
+
+os.environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "hide"
```

Comment:
It was already there, this will hide a welcome message from PyGame. I moved it up because `from gym.envs import make, spec, register` will import gym.wrappers, which will import PyGame in human_rendering.py. So we need to set the environment variable before importing `gym.envs`. The `noqa`s are then required because flake8 will complain otherwise. But I guess this will all be solved by moving PyGame into the Wrapper.